qa/tasks/cephfs/test_nfs.py

```diff
@@ -14,7 +15,7 @@
 NFS_POOL_NAME = '.nfs'  # should match mgr_module.py
 
 # TODO Add test for cluster update when ganesha can be deployed on multiple ports.
-class TestNFS(MgrTestCase):
+class TestNFS(MgrTestCase, CephFSTestCase):
```

Comment:
inheriting the full class doesn't work either
```
2023-08-09T15:33:45.851 INFO:tasks.cephfs_test_runner:test_cluster_info (tasks.cephfs.test_nfs.TestNFS) ... ERROR
2023-08-09T15:33:45.852 INFO:tasks.cephfs_test_runner:
2023-08-09T15:33:45.852 INFO:tasks.cephfs_test_runner:======================================================================
2023-08-09T15:33:45.853 INFO:tasks.cephfs_test_runner:ERROR: test_cluster_info (tasks.cephfs.test_nfs.TestNFS)
2023-08-09T15:33:45.853 INFO:tasks.cephfs_test_runner:----------------------------------------------------------------------
2023-08-09T15:33:45.854 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2023-08-09T15:33:45.854 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_1c7f5f4949aae52e52d69742399b261751d96650/qa/tasks/cephfs/test_nfs.py", line 39, in setUp
2023-08-09T15:33:45.854 INFO:tasks.cephfs_test_runner:    super(TestNFS, self).setUp()
2023-08-09T15:33:45.855 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_1c7f5f4949aae52e52d69742399b261751d96650/qa/tasks/cephfs/cephfs_test_case.py", line 152, in setUp
2023-08-09T15:33:45.855 INFO:tasks.cephfs_test_runner:    if len(self.mds_cluster.mds_ids) < self.MDSS_REQUIRED:
2023-08-09T15:33:45.856 INFO:tasks.cephfs_test_runner:AttributeError: 'NoneType' object has no attribute 'mds_ids'
2023-08-09T15:33:45.856 INFO:tasks.cephfs_test_runner:
2023-08-09T15:33:45.856 INFO:tasks.cephfs_test_runner:----------------------------------------------------------------------
```
So we're left with either going back to old behavior or @rishabh-d-dave 's patch.